src/dtos/dataset-dto.ts

```diff
@@ -24,7 +26,9 @@ export class DatasetDTO {
     datasetInfo: DatasetInfoDTO[];
     providers: DatasetProviderDTO[];
     topics: DatasetTopicDTO[];
-    team_id?: string;
+    team?: TeamDTO[];
+    start_date?: Date | null;
+    end_date?: Date | null;
```

Comment:
Do these need to be `| null` rather than just optional?

Reply:
Yes... Where I've added "null" its because you can't just set these to undefined for TypeORM to take the entries out of the database.  Things should generally exist or be `null` for TypeORM and for JavaScript Null and Undefined are of course different states of being.  So where I've added `| null` its because I need to be able to unset these values.